src/onegov/election_day/formats/election/wabstic_proporz.py

```diff
@@ -283,9 +242,10 @@ def import_election_wabstic_proporz(
 
         # Check if the entity is counted
         try:
-            entity['counted'] = False if int(line.sperrung or 0) == 0 else True
-        except ValueError:
-            line_errors.append(_("Invalid entity values"))
+            counted = validate_integer(line, 'sperrung')
+            entity['counted'] = False if counted == 0 else True
```

Comment:
I find this somewhat puzzling: Basically you say `entity['counted']` is `False` if `counted` is `0`.

This could be rewritten as follows: `entity['counted'] = not counted`. Which I think clarifies why this seems weird. Could this be a big? If not, can you add a comment to explain this?

Reply:
So `int(line.sperrung or 0)` corresponds 1:1 to `validate_integer(line, 'sperrung')` because `validate_integer` returns 0 if in the csv there was '' since `treat_emtpy_as_default=True` and `default` is 0 =) are the defaults of the function.
`False if counted == 0 else True` will thus also be the equivalent of the previous expression `False if int(line.sperrung or 0) == 0 else True`.
And you are right, `not counted` also is equivalent, so I will use this.

Reply:
`not counted` was just meant to illustrate how weird this is going to be for someone reading this in the future. I think you should add a good comment for this, otherwise someone might be tempted to just invert this, thinking it's an error.

Reply:
I added a comment what this is from the wabstic doc and remaned the variables.
`locking_time = validate_integer(line, 'sperrung', default=False)`
 `entity['counted'] = False if not locking_time else True`